src/styles/images/index.md.njk

```diff
@@ -8,53 +8,214 @@ layout: layout-pane.njk
+
+## Icons
+
+Avoid using icons in most cases. People can understand a single icon to mean different things, so they can be confusing.
+
+Icons can be more useful in case working systems, where users are familiar with the interface and return to it frequently. In this context, they can help users to scan pages more quickly. In most cases it’s still helpful to include a visible text label alongside any icons.
+
+Do not use a single icon to represent more than one thing. For example, the search icon (magnifying glass) should only be used for search functionality, and not also for screen magnification.
+
+## Avoid images that contain text
+
+Generally, you should avoid using images that contain any sort of text. Use written content or other alternatives wherever possible.
+
+Any text inside an image will be difficult to read, especially for users that need to read text using:
+- screen readers
+- copy and paste
+- highlight
+- resize
+- recolour
+
+Remember to avoid using images for unnecessary decoration, even if it’s an image of text.
+
+### Show text as written content, instead of images
+
+It’s usually unnecessary to use an image to merely show some text (known as an ‘image of text’).
+
+Give the information as written content instead, or use other alternatives.
+
+Examples of unnecessary ‘images of text’ include images that show:
+- the contents of a post from social media
+- an excerpt from a document
+- key facts from a slide presentation
+ 
+If you do choose to use the image anyway, include written content nearby that conveys the same meaning and context.
+
+[Find out more about ‘images of text’](https://www.w3.org/WAI/tutorials/images/textual/#image-of-styled-text-with-decorative-effect) on the Web Accessibility Initiative website.
+
+### If text is not the focus of an image
+
+It’s okay to use an image that just happens to have text in it.
+
+So if you choose to use an image to visually show something, and there’s text in it that’s not meant to be the focus of attention, it’s fine to leave the text in the image.
+
+Examples of images that just happen to have text:
+- a photo of a library with book covers and signs in the background
+- a collage to promote UK tourism that includes the Brighton Palace Pier sign
+- an illustration of a Manchester street with store signs in the background
+
+Write alt text that explains the meaning and context of the visual. Where the visual is essential to the user journey, include written content nearby that conveys the same meaning and context.
+
+However, make sure the image actually benefits the user and is not just for decoration.
+
+### Essential text in images
+
+There are some rare cases where you can use an image that contains text.
+
+Text in an image is considered essential if:
+- the text provides an intentional meaning and context
+- removing the text would cause the image lose the intended meaning and context
```

Comment:
Should this be 'would cause the image to lose'?

Reply:
Another good catch!